static/app/views/issueList/filters.tsx

```diff
@@ -73,7 +73,7 @@ const SearchContainer = styled('div')`
 const StyledPageFilterBar = styled(PageFilterBar)`
   flex: 1;
   width: 100%;
-  max-width: 25rem;
+  max-width: 30rem;
```

Comment:
The project selector uses a custom ellipsizing function that puts the ellipsis in the middle of the slug instead of at the end:
<img width="290" alt="Screen Shot 2022-11-10 at 1 06 49 PM" src="https://user-images.githubusercontent.com/44172267/201205973-e5208c60-4b79-4927-8ba8-6601a9dec740.png">

The max character count is 30. The filter bar needs to be longer than `25em` to show the all 30 characters.